2021-12-14 10:13:10 ERROR (MainThread) [metno] Access to https://aa015h6buqvih86i1.api.met.no/weatherapi/locationforecast/2.0/complete returned error 'ClientConnectorError'
2021-12-14 10:13:16 ERROR (MainThread) [homeassistant.components.updater] Error requesting Home Assistant update data: Cannot connect to host www.home-assistant.io:443 ssl:default [Connect call failed ('172.67.68.90', 443)]
2021-12-14 11:11:10 ERROR (MainThread) [metno] Access to https://aa015h6buqvih86i1.api.met.no/weatherapi/locationforecast/2.0/complete returned error 'ClientConnectorError'
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_activity
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_pedometer_distance
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_pedometer_floors_ascended
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_pedometer_floors_descended
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_pedometer_steps
2021-12-14 11:38:04 ERROR (MainThread) [homeassistant.components.mobile_app.webhook] Refusing to update iPhone von Johannes Hoffmann non-registered sensor: 3018aaf7fefdb3a924f279d131ad1fe1c68e807a3e2c97589bb316f42246179d_pedometer_avg_active_pace
2021-12-14 12:43:47 ERROR (MainThread) [metno] Access to https://aa015h6buqvih86i1.api.met.no/weatherapi/locationforecast/2.0/complete returned error 'ClientConnectorError'
2021-12-14 12:48:51 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID kitchen_light already exists - ignoring light.kitchen
2021-12-14 12:50:32 ERROR (MainThread) [homeassistant.components.light] Platform mqtt does not generate unique IDs. ID kitchen_light already exists - ignoring light.kitchen
